<launch>
<group ns="scooter">

<!--	<node name="map_server_costmap" pkg="map_server" type="map_server"  args="$(find utown_plaza)/rsz_larger_my_create.yaml">
		<remap from="map" to="map_cost"/>
		<param name="frame_id" value="/scooter/map"/>
	</node> --> 
     <!-- **** cost map and path planning **** -->
     <!--<node name="ped_path_planner" pkg="ped_is_despot" type="path_planner" output="screen" >-->
     <node name="ped_path_planner" pkg="ped_pathplan" type="path_planner" output="screen" >
         <rosparam file="$(find ped_is_despot)/params.yaml" command="load" />
         <remap from="ped_path_planner/planner/plan" to="plan" />
		 <param name="ped_costmap/global_frame" value="/scooter/map"/>
		 <param name="ped_costmap/robot_base_frame" value="/scooter/base_link"/>
     </node>
</group>
</launch>
